Card/src/main/java/com/paypal/android/card/CardClient.kt

```diff
@@ -1,6 +1,20 @@
 package com.paypal.android.card
 
-import com.paypal.android.core.PaymentsClient
+import com.paypal.android.card.network.CardAPIClient
+import com.paypal.android.core.APIClient
+import com.paypal.android.core.PaymentsConfiguration
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 
-@Suppress("UnusedPrivateMember")
-class CardClient(paymentsClient: PaymentsClient)
+class CardClient(val configuration: PaymentsConfiguration) {
+
+    private val cardAPI: CardAPIClient = CardAPIClient(APIClient(configuration))
+
+    fun approveOrder(orderId: String, card: Card, completion: (CardResult) -> Unit) {
```

Comment:
on iOS I believe we're using caps when referencing `..ID` objects - id `orderID` instead of `orderId`. not a big deal, just wanted to bring it up since it was a discussion on iOS

Reply:
Yeah I know the caps `ID` is a bit of a convention on iOS. I'm down to achieve parity wherever possible.

Reply:
I think we agreed to use `ID` (all caps) for naming in Northstar.

Reply:
Yup it's `ID` now.